sklearn/tests/test_multioutput.py

```diff
@@ -561,3 +562,20 @@ class A(MultiOutputEstimator, MultiOutputRegressor):
 
     with pytest.warns(FutureWarning, match="is deprecated in version 0.22"):
         A(SGDRegressor(random_state=0, max_iter=5))
+
+
+class DummyRegressorWithFitParams(DummyRegressor):
+    def fit(self, X, y, sample_weight=None, **fit_params):
+        self._fit_params = fit_params
+        return super().fit(X, y, sample_weight)
+
+
+def test_test_multi_target_regression_with_fit_params():
```

Comment:
Since we are adding support for both regressors and classifiers, you could parametrize the test such that we test both meta-estimator.